packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts

```diff
@@ -120,7 +120,7 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
         usernameOrEmail,
         team,
         translation,
-        ctx,
```

Comment:
Is ctx non needed anymore in sendSignupToOrganizationEmail ?

Reply:
Nevermind. Just saw the refactor.

Reply:
Looking at the function `ctx` was only used to get the user's name. Since I wanted to reuse the function outside of tRPC I removed the `ctx` param and replaced it with `inviterName`